resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -204,51 +212,139 @@ class="tw-font-bold">(@{{ artworks.total }})</span>
+                                                                    ${route('dielo.nahlad', {id: artwork.id, width: 220})} 220w,
+                                                                    ${route('dielo.nahlad', {id: artwork.id, width: 300})} 300w,
+                                                                    ${route('dielo.nahlad', {id: artwork.id, width: 600})} 600w,
+                                                                    ${route('dielo.nahlad', {id: artwork.id, width: 880})} 800w`"
+                                                    sizes="(max-width: 768px) 250vw, 100vw" />
+                                            </a>
+                                            <div :class="[{'tw-hidden': ic.isLoaded }, 'tw-w-full tw-saturate-50 tw-bg-gray-300']"
+                                                :style="{
+                                                                    'aspect-ratio': artwork.content.image_ratio || 1,
+                                                                    'background-color': artwork.content.hsl[0]
+                                                                        ? `hsl(${artwork.content.hsl[0].h}, ${artwork.content.hsl[0].s}%, ${artwork.content.hsl[0].l}%)`
+                                                                        : undefined,
+                                                                }"></div>
+                                        </div>
+                                    </catalog.artwork-image-controller>
+                                    <div class="tw-mt-6 tw-flex ">
+                                        <div class="tw-flex tw-flex-col tw-grow">
+                                            <a :href="route('dielo', {id: artwork.id})"
+                                                class="tw-pb-2 tw-text-lg tw-font-light tw-italic tw-leading-5">@{{ artwork.content.author[0] }}</a>
+                                            <a :href="route('dielo', {id: artwork.id})"
+                                                class="tw-pb-2 tw-text-lg tw-font-medium tw-leading-5">@{{ artwork.content.title }}</a>
+                                            <a :href="route('dielo', {id: artwork.id})"
+                                                class="tw-pb-2 tw-text-base tw-font-normal tw-leading-5">@{{ artwork.content.dating }}</a>
+                                        </div>
+                                        <div class="tw-flex tw-items-start tw-gap-4">
+                                            <favourite-controller v-slot="{ store }">
+                                                <button @click="store.toggleItem(artwork.id)">
+                                                    <span v-if="store.hasItem(artwork.id)">
+                                                        <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
probably no need for the spans here -- just `v-if` the svg
```suggestion
                                                    <svg v-if="store.hasItem(artwork.id)" xmlns="http://www.w3.org/2000/svg"
```